.env.local

```diff
@@ -1,3 +1,5 @@
+CLUSTERED_MODE=false
```

Comment:
There is a variable "CLUSTERED" in this file, I think we can remove it like in .env.test

Reply:
Thanks, missed that 👍